docs/content/workflows/pipelines/configuration-file.md

```diff
@@ -130,21 +130,21 @@ stages:
 You can export a full configuration of your pipeline with the CDS CLI using the `export` subcommand:
 
 ```bash
-cds pipeline export PROJECT_KEY pipeline_name
+cdsctl pipeline export PROJECT_KEY pipeline_name
```

Comment:
Usage of CLI is already here:  https://ovh.github.io/cds/cli/cdsctl/pipeline/export/
We should remove this part from his file and put a link instead